### DISCUSSION TITLE
Official python bindings

### DISCUSSION THREAD

**Discussion by juanmirocks** [swc-project/swc#9763] [Q&A]
### Describe the feature

Are there plans and/or could you use a hand to provide official python bindings of the SWC library? 

As far as I know, doing it with [pyo3](https://github.com/PyO3/pyo3) (& [maturin](https://github.com/PyO3/maturin)) should be easy.

### Additional context

My use case: I would like to have a fast _javascript to AST parser_ in python. I'm fine with a JSON output of the AST.

Other users had similar needs:
* https://github.com/swc-project/swc/issues/8065
* https://github.com/swc-project/swc/issues/1169

**Comment by kwonoj** [comment]
There's no plan for having official bindings for other languages so far.

**Reply by juanmirocks** [reply]
Thank you @kwonoj for your confirmation.
